llvm/lib/CodeGen/MIRParser/MIParser.cpp

```diff
@@ -2547,6 +2547,16 @@ bool MIParser::parseCFIOperand(MachineOperand &Dest) {
     CFIIndex = MF.addFrameInst(MCCFIInstruction::createLLVMDefAspaceCfa(
         nullptr, Reg, Offset, AddressSpace, SMLoc()));
     break;
+  case MIToken::kw_cfi_llvm_reg_offset: {
+    Register FrameReg;
+    if (parseCFIRegister(Reg) || expectAndConsume(MIToken::comma) ||
+        parseCFIRegister(FrameReg) || expectAndConsume(MIToken::comma) ||
+        parseCFIOffset(Offset))
+      return true;
+    CFIIndex = MF.addFrameInst(
+        MCCFIInstruction::createLLVMRegOffset(nullptr, Reg, FrameReg, Offset));
```

Comment:
FrameReg here is a Register but createLLVMRegOffset takes it as an unsigned. I'm slowly trying to remove implicit conversions from Register to unsigned. Please propagate Register or used Register::id()

Reply:
Oh this is a mistake in parseCFIRegister interface. FrameReg and Reg should not be `Register`. I'll fix that.